### DISCUSSION TITLE
GPU mining?

### DISCUSSION THREAD

**Discussion by Flight1530** [revoxhere/duino-coin#426] [Ideas]
I have older computers that I run, will there be any GPU (old cards) mining to Duino Coin or a say a USB miner for the RasperyPi as a addon?

**Comment by JoyBed** [comment]
Yes, im working on it.

**Reply by Flight1530** [reply]
Thanks.

**Comment by revoxhere** [comment]
It may be possible once such a miner is crated, but don't expect good earnings from it

**Comment by BastelPichi** [comment]
> I have older computers that I run, will there be any GPU (old cards) mining to Duino Coin or a say a USB miner for the RasperyPi as a addon?

You can just use an arduino as addon. Theres also a youtuber on the discord whos designing a usb miner style arduino....
